org.scala-ide.play2/src/org/scalaide/play2/templateeditor/processing/TemplateProcessingProvider.scala

```diff
@@ -0,0 +1,24 @@
+package org.scalaide.play2.templateeditor.processing
+
+import org.eclipse.core.runtime.RegistryFactory
+import org.scalaide.play2.PlayPlugin
+import org.scalaide.play2.properties.PlayPreferences
+
+object TemplateProcessingProvider {
+  val ExtensionPointId = "org.scalaide.play2.template.processing"
+
+  def templateProcessing(): TemplateProcessing = {
```

Comment:
Loading extensions is not safe, since users can theoretically add a broken one. I would suggest to load them in a safe runner and also to not expose the class. See how it is done here: https://github.com/scala-ide/scala-ide/blob/master/org.scala-ide.sdt.core/src/org/scalaide/core/internal/quickassist/QuickAssistMapping.scala